fixtures/sub-process-no-killable

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env node
```

Comment:
Sorry to bother, but I'm working on #170 and I might need to name a fixture exactly like this one; I will need to implement a similar retry as yours, but for killing the tree of the descendants of the spawned process. Also the prefix `sub-process` is used already for the fixtures that spawn themselves a sub-process.

Could you rename this to something without `sub-process` prefix? If I might suggest `unkillable` sounds good as a single name, or `no-killable` as well; your choice, I'll follow on my PR.

Reply:
Hi, thanks for letting me know. It's called `no-killable` now.